src/main/kotlin/org/ionproject/integration/utils/Try.kt

```diff
@@ -32,26 +35,28 @@ sealed class Try<out T> {
         override fun <R> match(success: (T) -> R, error: (Exception) -> R) = success(value)
     }
 
-    class Error(val e: Exception) : Try<Nothing>() {
+    class Error<out T>(val e: Exception) : Try<T>() {
```

Comment:
Why this change?

Reply:
I've made the change while trying to implement mapError and forgot to change it back.